mastodon.yaml

```diff
@@ -71,6 +76,50 @@ storage:
       group:
         name: mastodon
 
+    # Monitoring
+    - path: /opt/mastodon/grafana
+      mode: 0755
+      user:
+        name: mastodon
+      group:
+        name: mastodon
+    - path: /opt/mastodon/grafana/data
+      mode: 0755
+      user:
+        name: mastodon
+      group:
+        name: mastodon
+    - path: /opt/mastodon/grafana/dashboards
+      mode: 0755
+      user:
+        name: mastodon
+      group:
+        name: mastodon
+    - path: /opt/mastodon/grafana/datasources
+      mode: 0755
+      user:
+        name: mastodon
+      group:
+        name: mastodon
+    - path: /opt/mastodon/prometheus
+      mode: 0755
+      user:
+        name: mastodon
+      group:
+        name: mastodon
+    - path: /opt/mastodon/prometheus/etc
```

Comment:
We can simplify the config by removing the directory creation for configuration files created below. Directories will be created by Ignition with the default permissions (`755`).

Reply:
This will create directories owned by root unfortunately. I'd like to have these owned by the local `mastodon` user.